src/apps/works/admin/options.py

```diff
@@ -0,0 +1,33 @@
+from anion.contrib import admin
+
+from ..models import Article, Book, Patent
+
+
+class WorkAdmin(admin.ModelAdmin):
+    """
+    Admin options common to all the models based on `works.Work`.
```

Comment:
Append "class" at the end of the docstring.

```suggestion
    Admin options common to all the models based on `works.Work` class.
```